src/modules/reportes_poai/reportes_poai.service.ts

```
@@ -0,0 +1,665 @@
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
+import { SupabaseService } from '../../config/supabase/supabase.service';
+import {
+  ReportePeriodoRequest,
```

ESLint

[error] 'ReportePeriodoRequest' is defined but never used.

(@typescript-eslint/no-unused-vars)